index.rb

```diff
@@ -144,6 +146,31 @@ def show(**kwargs)
   end
 end
 
+venues << Venue.new(:name => 'Knockout', :link => 'https://theknockoutsf.com') do
+  URI.open(URI.join(link, 'events/feed')) do |xml|
+    Nokogiri::XML(xml).css('item').map do |item|
+      # Nokogiri doesn't decode this correctly, so we have to force it :(
+      description = CGI.unescape_html(item.css('description').text)
+
+      # Workaround for the occasional empty title
+      title = item.css('title').text
+      title = description.split('•').first if title.empty?
+
+      next if title =~ /(karaoke|bingo)/i
+
+      show(
+        time: Time.at(
+          Time.parse(item.css('pubDate').text),
+          in: TZInfo::Timezone.get('America/Los_Angeles')
```

Comment:
Maybe should set `ENV['TZ']`?

Reply:
Yep, definitely. I removed that in favor of `Time#getlocal`